src/components/rewards-popup/index.test.tsx

```diff
@@ -18,12 +18,15 @@ describe('RewardsPopup', () => {
     return shallow(<RewardsPopup {...allProps} />);
   };
 
-  it('renders your rewards count in USD', function () {
-    const wrapper = subject({ usd: '$5.71' });
-
-    const skeleton = wrapper.find('.rewards-popup__rewards-usd SkeletonText');
-    expect((skeleton.prop('asyncText') as any).text).toEqual('$5.71');
-  });
+  /* @note 29-06-2023
```

Comment:
let's just actually delete this test rather than commenting out, and then 🚢

Reply:
done